gammapy/utils/fitting/tests/test_fit.py

```diff
@@ -97,3 +97,23 @@ def test_likelihood_profile():
 
     # Check that original value state wasn't changed
     assert_allclose(fit._model.parameters["x"].value, 2)
+
+
+def test_minos_contour():
+    fit = MyFit()
+    fit.optimize(backend="minuit")
+    result = fit.minos_contour("x", "y")
+
+    assert result["is_valid"] is True
+
+    x = result["x"]
+    assert_allclose(len(x), 10)
+    assert_allclose(x[0], 1, rtol=1e-5)
+    assert_allclose(x[-1], 1.499963, rtol=1e-5)
+    y = result["y"]
+    assert_allclose(len(y), 10)
+    assert_allclose(y[0], 300, rtol=1e-5)
+    assert_allclose(y[-1], 300.866004, rtol=1e-5)
+
```

Comment:
Maybe add one assert on the likelihood value at one of the contour points? But I guess it's not really needed...

Reply:
I'm not sure if the likelihood at these points is available, or if we should compute it ourselves. Certainly having the stat or ts val of the contour in the return dict would be great.